internal/sdks/sdk_instructions/python.md

```diff
@@ -0,0 +1,56 @@
+# Set up your application
+
+If you want to skip ahead, the final code is available in our [GitHub repository](https://github.com/launchdarkly/hello-python).
+
+
```

Comment:
I think maybe when the instructions are all in we might want to pull this intro text out of the markdown file/viewport box